### DISCUSSION TITLE
使用<el-image>，在vite2 build之后加载图片失败。

### DISCUSSION THREAD

**Discussion by klakesen** [element-plus/element-plus#2413] [Q&A]
`<el-image src="/src/views/system/login/image/img-password.png"></el-image>`
在npm run build后无法正常加载图片。
`<img src="/src/views/system/login/image/img-username.png"/>`
在npm run build后可以正常加载图片。
这个是element-plus的bug吗？
![gaitubao_截图20210701203248431_png](https://user-images.githubusercontent.com/15100627/124214056-333eee80-db24-11eb-8696-3c4b204d950a.png)

**Comment by iamkun** [comment]
please check https://github.com/element-plus/element-plus/issues/2277

**Reply by klakesen** [reply]
> please check #2277

vite version 2.3.8改成2.3.7，npm run build后依旧无法加载图片
